nym-vpn-core/crates/nym-gateway-probe/netstack_ping/gen.go

```diff
@@ -0,0 +1,296 @@
+package main
```

Comment:
Nit: can this file be generated instead of being committed to repo?

Reply:
It could be, but then we'd need to drag rust2go cli into the CI, so keeping it simple for now